distro/docker-compose/config/keycloak/microcks-realm.json.template

```diff
@@ -297,8 +295,28 @@
           "attributes": {}
         }
       ],
-      "microcks-app": [],
+      "microcks-serviceaccount": [],
+      "microcks-app-js": [
+        {
+          "id": "d97e2d8b-cd4b-42df-b7d3-a55c1de5cff2",
+          "name": "user",
+          "composite": false,
+          "clientRole": true,
+          "containerId": "27bc63fd-1780-4acd-992a-48c2ef2f429f",
+          "attributes": {}
+        }
+      ],
       "security-admin-console": [],
+      "microcks-app": [
+        {
+          "id": "b70db266-0a25-48b8-a53f-5666febd06ea",
+          "name": "user",
+          "composite": false,
+          "clientRole": true,
+          "containerId": "61397ec1-d0fa-4b9b-a38a-c301326b7833",
+          "attributes": {}
+        }
```

Comment:
To be fully aligned, we should also add the `manager` and `admin` roles as in https://github.com/microcks/microcks/blob/master/install/docker-compose/keycloak-realm/microcks-realm-sample.json#L65